scram-tools.file/tools/cepgen/cepgen.xml

```diff
@@ -0,0 +1,13 @@
+<tool name="cepgen" version="@TOOL_VERSION@">
+  <lib name="CepGen"/>
+  <lib name="CepGenHepMC2"/>
+  <lib name="CepGenHepMC3"/>
+  <lib name="CepGenLHAPDF"/>
+  <lib name="CepGenPythia6"/>
+  <client>
+    <environment name="CEPGEN_PATH" default="@TOOL_ROOT@"/>
```

Comment:
@forthommel , sorry I missed this, cms convention is to use `<toolname>_BASE` so can you please release all `CEPGEN_PATH`  with `CEPGEN_BASE`

Also if `cepgen` executable is built by default then add 
```
<runtime name="PATH" value="$CEPGEN_BASE/bin" type="path"/>
```
so that it is available in cmssw env

Reply:
Actually, `CEPGEN_PATH` is used internally by cepgen to define where to load its various external files (e.g. MCD files, interpolation grids). I could specify the path manually in the EDProducer though, but that might complicate things?

Reply:
@forthommel , if you need `CEPGEN_PATH` env variable to be set at runtime then please use
```
<runtime name="CEPGEN_PATH" value="$CEPGEN_BASE/share/CepGen"/>
```
see other toolfiles ( e.g. https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_12_5_X/master/scram-tools.file/tools/root/root_interface.xml ) on how to set runtime env.

Also as I mentioned here https://github.com/cms-sw/cmsdist/pull/8023#discussion_r948945815, please change `CEPGEN_DIR` to `CEPGEN_BASE` every where in toolfile